Makefile

```diff
@@ -48,8 +48,19 @@ docs: .clone-st2 requirements .requirements-st2 .docs
 PHONY: .docs
 .docs: .community-docs
 
+PHONY: .remove-bwc-solutions-folders
```

Comment:
Nit: Usually these targets are called "clean" targets in Makefile lingo. So .clean-bwc-solutions-folders might be a better name.